pandas/tests/frame/indexing/test_setitem.py

```diff
@@ -680,6 +680,20 @@ def test_boolean_mask_nullable_int64(self):
         )
         tm.assert_frame_equal(result, expected)
 
+    def test_2d_array(self):
```

Comment:
test_setitem_npmatrix_2d; this location is fine

Reply:
:+1: